addons/mail/static/src/components/attachment_image/attachment_image_tests.js

```diff
@@ -0,0 +1,97 @@
+/** @odoo-module **/
+
+import AttachmentImage from '@mail/components/attachment_image/attachment_image';
+
+const components = { AttachmentImage };
+
+import {
+    afterEach,
+    beforeEach,
+    createRootComponent,
+    start,
+} from '@mail/utils/test_utils';
+
+QUnit.module('mail', {}, function () {
+QUnit.module('components', {}, function () {
+QUnit.module('attachment_image', {}, function () {
+QUnit.module('attachment_image_tests.js', {
+    beforeEach() {
+        beforeEach(this);
+
+        this.createAttachmentImageComponent = async (attachment, otherProps) => {
+            const props = Object.assign({ attachmentLocalId: attachment.localId }, otherProps);
+            await createRootComponent(this, components.AttachmentImage, {
+                props,
+                target: this.widget.el,
+            });
+        };
+
+        this.start = async params => {
+            const { env, widget } = await start(Object.assign({}, params, {
+                data: this.data,
+            }));
+            this.env = env;
+            this.widget = widget;
+        };
+    },
+    afterEach() {
+        afterEach(this);
+    },
+});
+
+QUnit.test('auto layout with image', async function (assert) {
+    assert.expect(7);
+
+    await this.start();
+    const attachment = this.env.models['mail.attachment'].create({
+        filename: "test.png",
+        id: 750,
+        mimetype: 'image/png',
+        name: "test.png",
+    });
+
+    await this.createAttachmentImageComponent(attachment, {
+        isEditable: false,
+    });
+    assert.strictEqual(
+        document.querySelectorAll(`
+            .o_AttachmentImage_details:not(.o_AttachmentImage_imageOverlayDetails)
+        `).length,
+        0,
+        "attachment should not have a details part directly"
+    );
```

Comment:
assert doesNotHaveClass (or containsNone, I'm not exactly sure what it is trying to check)